### DISCUSSION TITLE
How to load an image only from Cache (not from network)?

### DISCUSSION THREAD

**Discussion by vijayendra-tripathi** [coil-kt/coil#1214] [Q&A] [answered]
I wish to read image instantly from cache if not available then I may request the image later from network. How can I use coil to only read from cache and not from network?

**Comment by newmanw** [comment]
@vijayendra-tripathi did you find a solution?

**Comment by Susheelkaram** [comment, marked as answer]
I think you can achieve this by setting `networkCachePolicy(CachePolicy.DISABLED)`

**Comment by Liangwenb** [comment]
respectCacheHeaders(false)
